### ISSUE TITLE
BPM black bars on the sides with aspect ratio wider than 16:9

### ISSUE THREAD

**Issue by eathtespagheti** [ValveSoftware/steam-for-linux#7486]
#### Your system information

* Steam client version (build number or date): Nov 20 2020
* Distribution (e.g. Ubuntu): Arch Linux
* Opted into Steam client beta?: Yes
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
Resolution in BPM it's locked to 16:9 aspect ratio, so displaying BPM in any wider resolution results in black bars to the sides

#### Steps for reproducing this issue:

1. Use a resolution with an aspect ration wider than 16:9 (e.g. 21:9)
2. Launch BPM
3. Enjoy side black bars

**Comment by kisak-valve** [comment]
Hello @eathtespagheti, this sounds like the intended behavior so that there aren't any aspect ratio related layout issues with Big Picture Mode. Let's treat this issue report as a feature request to support more aspect ratios.

**Comment by eathtespagheti** [comment]
yeah effectively it's more of a feature request, thanks for the fast reply and help!

**Comment by legacychimera247** [comment]
Hello, so it would it be hard to support 16:10?because i hate those black bars...
